src/server/rest/CentralRestServer.ts

```diff
@@ -99,15 +99,15 @@ export default class CentralRestServer {
 
   startSocketIO(): void {
     // Log
-    const logMsg = 'Starting REST SocketIO Server...';
+    const logMsg = 'Starting REST SocketIO Server';
     Logging.logInfo({
       tenantID: Constants.DEFAULT_TENANT,
       module: MODULE_NAME, method: 'startSocketIO',
       action: ServerAction.STARTUP,
-      message: logMsg
+      message: logMsg + '...'
     });
     // eslint-disable-next-line no-console
-    console.log(`Starting REST SocketIO Server ${cluster.isWorker ? 'in worker ' + cluster.worker.id : 'in master...'}`);
+    console.log(`${logMsg} ${cluster.isWorker ? 'in worker ' + cluster.worker.id.toString() : 'in master...'}`);
```

Comment:
No need this kind of optimization, I want to read the message without looking for a var

Reply:
It's 10 lines up ;)

Reply:
> It's 10 lines up ;)

Ok for that one LOL.